README.md

```diff
@@ -319,15 +319,15 @@ For Kubernetes variants in VMware, you must specify:
 
 The following settings can be optionally set to customize the node labels and taints. Remember to quote keys (since they often contain ".") and to quote all values.
 * `settings.kubernetes.node-labels`: [Labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/) in the form of key, value pairs added when registering the node in the cluster.
-* `settings.kubernetes.node-taints`: [Taints](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/) in the form of key, value and effect entries added when registering the node in the cluster.
+* `settings.kubernetes.node-taints`: [Taints](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/) in the form of key, values and effects entries added when registering the node in the cluster.
   * Example user data for setting up labels and taints:
     ```
     [settings.kubernetes.node-labels]
     "label1" = "foo"
     "label2" = "bar"
     [settings.kubernetes.node-taints]
-    "dedicated" = "experimental:PreferNoSchedule"
-    "special" = "true:NoSchedule"
+    "dedicated" = ["experimental:PreferNoSchedule", "experimental:NoExecute"]
+    "special" = ["true:NoSchedule"]
```

Comment:
nit: Do we want to leave one of these as a String to demonstrate that it can also be parsed?

Reply:
Good idea. Though I'm not sure if people would be confused seeing two different ways of specifying a node taint. Maybe we should just deprecate the old way of mapping single keys to single values, and encourage users to use the new representation.